internal/tests/cluster/worker_bytesupdown_test.go

```diff
@@ -0,0 +1,131 @@
+package cluster
+
+import (
+	"context"
+	"net"
+	"testing"
+
+	"github.com/hashicorp/boundary/api/sessions"
+	"github.com/hashicorp/boundary/api/targets"
+	"github.com/hashicorp/boundary/internal/cmd/config"
+	"github.com/hashicorp/boundary/internal/daemon/controller"
+	"github.com/hashicorp/boundary/internal/daemon/worker"
+	"github.com/hashicorp/boundary/internal/tests/helper"
+	"github.com/hashicorp/dawdle"
+	"github.com/hashicorp/go-hclog"
+	"github.com/stretchr/testify/require"
+)
+
+func TestWorkerBytesUpDown(t *testing.T) {
+	t.Parallel()
+	require := require.New(t)
+
+	logger := hclog.New(&hclog.LoggerOptions{
+		Name:  t.Name(),
+		Level: hclog.Trace,
+	})
+
+	conf, err := config.DevController()
+	require.NoError(err)
+
+	pl, err := net.Listen("tcp", "localhost:0")
+	require.NoError(err)
+	c1 := controller.NewTestController(t, &controller.TestControllerOpts{
+		Config:                    conf,
+		InitialResourcesSuffix:    "1234567890",
+		Logger:                    logger.Named("c1"),
+		PublicClusterAddr:         pl.Addr().String(),
+		StatusGracePeriodDuration: helper.DefaultGracePeriod,
+	})
+	defer c1.Shutdown()
```

Comment:
Nit: use `t.Cleanup` for this sort of thing in tests, and don't forget to check the errors!

Reply:
Fixed in [88742e5](https://github.com/hashicorp/boundary/pull/2503/commits/88742e510ec70d1934256b6323dbdd667ed28e09)